datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JsonWebKey.kt

```diff
@@ -281,10 +282,10 @@ data class JsonWebKey(
      * (i.e. if all key params are set), or the first error.
      */
     fun toCryptoPublicKey(): KmmResult<CryptoPublicKey> =
-        runCatching {
+        catching {
```

Comment:
This `catching` can probably moved up one line, to reduce indendation in the body